prov/verbs/src/fi_verbs.c

```diff
@@ -445,5 +494,15 @@ VERBS_INI
 				 "greater", &verbs_default_inline_size))
 		return NULL;
 
+	if (fi_ibv_get_param_int("min_rnr_timer", "Set min_rnr_timer QP "
+				 "attribute (0 - 31)", &verbs_min_rnr_timer))
+		return NULL;
+
+	if (verbs_min_rnr_timer > 31) {
+		VERBS_WARN(FI_LOG_CORE, "Invalid value for min_rnr_timer."
+			   "Valid range: 0 - 31 \n");
+		return NULL;
+	}
```

Comment:
Most of these are changing some default value, that is later adjusted based on hardware limits anyway.  So my vote is to attempt to continue, rather than having an error be a showstopper.